freqtrade/rpc/rpc.py

```diff
@@ -316,8 +316,7 @@ def _exec_forcesell(trade: Trade) -> None:
                         and order['side'] == 'buy':
                     exchange.cancel_order(trade.open_order_id, trade.pair)
                     trade.close(order.get('price') or trade.open_rate)
-                    # TODO: sell amount which has been bought already
-                    return
+                    trade.amount = order['filled']
```

Comment:
are you sure "filled" is always part of the response?
it's not in dry-mode - and i assume it's also not depending on the exchange used - so i think there should be a check if it exists in the order-dict...